Author: Andreas Beckmann <anbe@debian.org>
Description: fix typos

--- a/lib/SPIRV/SPIRVLowerOCLBlocks.cpp
+++ b/lib/SPIRV/SPIRVLowerOCLBlocks.cpp
@@ -33,7 +33,7 @@
 //===----------------------------------------------------------------------===//
 //
 // SPIR-V specification doesn't allow function pointers, so SPIR-V translator
-// is designed to fail if a value with function type (except calls) is occured.
+// is designed to fail if a value with function type (except calls) is occurred.
 // Currently there is only two cases, when function pointers are generating in
 // LLVM IR in OpenCL - block calls and device side enqueue built-in calls.
 //
@@ -126,7 +126,7 @@ char SPIRVLowerOCLBlocksLegacy::ID = 0;
 
 INITIALIZE_PASS(
     SPIRVLowerOCLBlocksLegacy, "spv-lower-ocl-blocks",
-    "Remove function pointers occured in case of using OpenCL blocks", false,
+    "Remove function pointers occurred in case of using OpenCL blocks", false,
     false)
 
 llvm::ModulePass *llvm::createSPIRVLowerOCLBlocksLegacy() {
